cdc-debezium-app-dependencies/pom.xml

```diff
@@ -13,17 +13,19 @@
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
-		<version>2.1.3.RELEASE</version>
+		<version>2.1.7.RELEASE</version>
 		<relativePath/>
 	</parent>
 
 	<properties>
-		<spring-cloud-stream.version>2.1.2.RELEASE</spring-cloud-stream.version>
-		<version.debezium>0.9.5.Final</version.debezium>
-		<!-- Note: postgresql version MUST match the version used by
-		the debezium postgres connector -->
+<!--		<spring-cloud-stream.version>2.1.2.RELEASE</spring-cloud-stream.version>-->
+		<version.debezium>0.10.0.Final</version.debezium>
```

Comment:
Any chances that we can upgrade already: https://debezium.io/blog/2019/12/18/debezium-1-0-0-final-released/ ?

Reply:
No

Reply:
it took 1 year to get to this point. The upgrade will come with next version :) probably after an year :)